cli/dcoscli/marathon/main.py

```diff
@@ -874,22 +874,40 @@ def task_kill(self, task_ids, scale, wipe):
         :type scale: bool
         :param wipe: whether remove reservations and persistent volumes.
         :type wipe: bool
+        :param json_: output JSON if true
+        :type json_: bool
         :returns: process return code
         :rtype: int
         """
 
         client = self._create_marathon_client()
         payload = client.kill_and_scale_tasks(task_ids, scale, wipe)
 
-        # No deployment was started and no tasks were killed
+        def print_deployment(deployment, json_):
+            if json_:
+                emitter.publish('Created deployment:')
+                emitter.publish(deployment)
```

Comment:
if json we should just print `deployment` as is, and remove the `Created deployment` line